patches/chromium/.patches

```diff
@@ -121,3 +121,4 @@ fix_on-screen-keyboard_hides_on_input_blur_in_webview.patch
 preconnect_manager.patch
 fix_remove_caption-removing_style_call.patch
 build_allow_electron_to_use_exec_script.patch
+revert_use_accessibility_pkey_when_setting_page_access.patch
```

Comment:
generally when there's an upstream patch for something that breaks us i prefer to cherry-pick the upstream patch into our branch rather than revert the breakage. that way the patch is automatically removed when we rebase onto the next roll.

Reply:
@nornagon I agree, but that CL hasn’t been merged yet and is dependent on other CLs

Reply:
ah, okay. dependent on other CLs makes sense :) i don't mind cherry-picking as-yet-unmerged things.